atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/Transaction.java

```diff
@@ -51,7 +51,8 @@
     Map<Cell, byte[]> get(TableReference tableRef, Set<Cell> cells);
 
     /**
-     * Creates a visitable that scans the provided range.
+     * Creates a visitable that scans the provided range.  This method retrieves all cell-value pairs
+     * spanned by the <code>RangeRequest</code>.
```

Comment:
This is not the right place. This extra text doesn't really add anything.
You should be putting JavaDoc on `BatchingVisitableView#immutableCopy` saying that this will return a `Set` or `List` with all objects in the range, not just the batch.